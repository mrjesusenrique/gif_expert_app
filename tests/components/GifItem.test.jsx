const { render } = require("@testing-library/react");
import GifItem from "../../src/components/GifItem";

describe("Pruebas en <GifItem />", () => {
  const title = "Leones";
  const url = "http://leones.com";
  const id = "12345";

  test("Debe hacer hacer match con el snapshot", () => {
    const { container } = render(<GifItem title={title} url={url} id={id} />);
    expect(container).toMatchSnapshot();
  });
});
