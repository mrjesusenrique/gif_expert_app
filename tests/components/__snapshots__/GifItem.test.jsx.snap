// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem /> Debe hacer hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
    id="12345"
  >
    <img
      alt="Leones"
      src="http://leones.com"
    />
    <p>
      Leones
    </p>
  </div>
</div>
`;
